runtime/lua/vim/keymap.lua

```diff
@@ -36,14 +36,16 @@ local keymap = {}
 ---@param lhs string           Left-hand side |{lhs}| of the mapping.
 ---@param rhs string|function  Right-hand side |{rhs}| of the mapping. Can also be a Lua function.
 --
----@param opts table A table of |:map-arguments| such as "silent". In addition to the options
----                  listed in |nvim_set_keymap()|, this table also accepts the following keys:
----                  - buffer: (number or boolean) Add a mapping to the given buffer. When "true"
----                    or 0, use the current buffer.
----                  - remap: (boolean) Make the mapping recursive. This is the
----                  inverse of the "noremap" option from |nvim_set_keymap()|.
----                  Default `false`.
----                  - replace_keycodes: (boolean) defaults to true if "expr" is true.
+---@param opts table A table of |:map-arguments|. Accepts all options accepted by the {opts}
+---                  parameter in |nvim_set_keymap()|, with the following notable differences in
+---                  default values:
+---                  - replace_keycodes: (boolean) Defaults to true if "expr" is true.
+---                  In addition to those options, the table accepts the following keys:
```

Comment:
I previously also tried to insert a sentence after a list in arguments, but it didn't seem to work. As you can see in the generated docs this is appended to the previous line.

Reply:
Oh, this seems to work:
```suggestion
---                  .
---                  In addition to those options, the table accepts the following keys:
```